src/App.css

```diff
@@ -33,6 +32,44 @@
   color: #61dafb;
 }
 
+.Background-logo {
+  background-color: darkred;
+  background-image: url('./Background.jpg');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.Start-button {
```

Comment:
Did we use it somewhere?

Reply:
Background-size is used for proper image display. Start-button is not and will be deleted.